web/satellite/src/utils/converter.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+const KB = 1e3;
+const MB = 1e6;
+const GB = 1e9;
+const TB = 1e12;
+const PB = 1e15;
+
+/**
+ * Used to display correct and convenient data
+ */
+export class Converter {
```

Comment:
Converter is to abstract name. Lets name it Size, for example.
Look how similar functionality was implemented on backend
https://github.com/storj/storj/blob/master/private/memory/size.go